Fixing Typo of chapter 6 put_it_together_two

aniket866 opened the pull request:
## Pull Request checklist

Please review the below PR requirements:

- [x] Build was run locally and any changes were pushed
- [x] Ensure that the title clearly describes the changes
- [x] Issue to be referenced in the PR description rather than in the commits or PR title
- [x] Clean commit history

## How to Reproduce:https://savingsatoshi.com/en/chapters/chapter-6/put-it-together-2-normal
     - Then go to step 3 and see the Typo `sign_input(index, key)`. 

## This PR fixes the Typo mentioned in the issue #1375 

### Should be `sign_input(index, priv, pub, sighash) `not the current `sign_input(index, key).`

@pinheadmz Please review this and can be merged , Closes #1375

aniket866 committed:
Fixing Typo of chapter 6 put_it_together_two

pinheadmz reviewed (approved):
ACK, thanks.
This is just an inconsistent documentation fix with no behavioral changes and it is correct

satsie commented:
Thank you Zip for the review! And thank you @aniket866 for the PR! I will merge it now and it will go out in the next release. Do you know if the other locales need updating? de.ts, es.ts, hi.ts, ja.ts, nl.ts, vi.ts, and zh-cn.ts?

aniket866 commented:
@satsie  yes, In all other locales only some chapters are implemented in that Locale language, for most of the locales            (except English) there is only place holders (like this in the screenshot), these Chapters are still using English text contents on their page. 

Recently, I started debugging those locales and I have corrected and implemented missings of Hindi locale, It was previously using English content on Hindi pages. Now I am working on other Locales to fix them too.

for now these PR #1372 #1373 Completely fixes all the inconsistencies of Hindi Locales. Please have a look.
